### ISSUE TITLE
romote play

### ISSUE THREAD

**Issue by NSGG404** [SteamTools-Team/SteamTools#145]
Would you take remote play on steam into consideration,please?

**Comment by rmbadmin** [comment]
Yes, I wish add support for remote play, but it requires server support, we don't have at the moment server.
